app/backend/src/couchers/servicers/conversations.py

```diff
@@ -342,20 +342,32 @@ def SearchMessages(self, request, context):
             )
 
     def CreateGroupChat(self, request, context):
-        if len(request.recipient_user_ids) < 1:
+        with session_scope() as session:
+            recipient_user_ids = [
+                user.id
+                for user in (
+                    session.query(User.id).filter(User.is_visible).filter(User.id.in_(request.recipient_user_ids)).all()
+                )
+            ]
+
+        # make sure all requested users are visible
+        if len(recipient_user_ids) != len(request.recipient_user_ids):
+            context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.USER_NOT_FOUND)
+
+        if len(recipient_user_ids) < 1:
```

Comment:
Know it was wrong before as well, but this should just be `if not recipient_user_ids`

Reply:
You mean

`if not len(recipient_user_ids) == len(request.recipient_user_ids):`

?

Reply:
No, this line is checking if the `recipient_user_ids` list is empty or not. The pythonic way is to check whether the list is "falsey", i.e. whether `not recipient_user_ids` evaluates to `True`, which happens if and only if `recipient_user_ids` is an empty list.

Reply:
Got it. I'll change it. But ya, unrelated to this PR.